### PR TITLE
Remove importCommunity from QML and Nim code

### PR DESCRIPTION
**Pull request by elizasorber**
Handling issue #14310. Removed functions associated with the API.

### What does the PR do
Removed functions associated with the API.

### Affected areas
Stores code

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

### Cool Spaceship Picture

<!-- optional but cool ->

### CODE REVIEW

File: src/app/modules/main/communities/module.nim [hunk 1]
```diff
@@ -430,16 +419,6 @@ method communityDataImported*(self: Module, community: CommunityDto) =
 method communityInfoRequestFailed*(self: Module, communityId: string, errorMsg: string) =
   self.view.emitCommunityInfoRequestCompleted(communityId, errorMsg)
 
-method importCommunity*(self: Module, communityId: string) =
-  self.view.emitImportingCommunityStateChangedSignal(communityId, ImportCommunityState.ImportingInProgress.int, errorMsg = "")
-  self.controller.importCommunity(communityId)
-
-method onImportCommunityErrorOccured*(self: Module, communityId: string, error: string) =
-  self.view.emitImportingCommunityStateChangedSignal(communityId, ImportCommunityState.ImportingError.int, error)
```
Comment (jrainville) [inline src/app/modules/main/communities/module.nim:438]: This one is still needed for the event.

**Review by jrainville — COMMENTED** [review]
Nice job. Some of the events and functions are still used with the new API. I pointed them out in comments. Otherwise it looks good.

**Comment by elizasorber** [comment]
I updated my pr with your suggested changes! Lmk if anything else looks wrong and I'll fix it.

**Review by jrainville — COMMENTED** [review]
One small thing

**Review by jrainville — APPROVED** [review]
Sorry for the delay to re-review, I didn't see the last commit.

**Review by endulab — APPROVED** [review]

**Comment by jrainville** [comment]
Now, how do we make CI pick it up :thinking: cc @jakubgs

**Comment by caybro** [comment]
> Now, how do we make CI pick it up 🤔 cc @jakubgs

It's a fork... I don't think it's gonna work this way

**Comment by jrainville** [comment]
> > Now, how do we make CI pick it up 🤔 cc @jakubgs
> 
> It's a fork... I don't think it's gonna work this way

Yeah that's what I thought. I guess I'll have to cherry-pick and create another PR

**Comment by jrainville** [comment]
I rebased and unforked this branch/PR here: https://github.com/status-im/status-desktop/pull/14814

Closing this one in favor of the new one.

**Pull request closed without merging**